include/pfnet/bus.h

```diff
@@ -18,8 +18,10 @@
 #include "uthash.h"
 
 // Limits
-#define BUS_DEFAULT_V_MAX 1.1 /**< @brief Default maximum voltage magnitude. */
-#define BUS_DEFAULT_V_MIN 0.9 /**< @brief Default minimum voltage magnitude. */
+#define BUS_DEFAULT_V_MAX_REG 1.1 /**< @brief Default maximum regulated voltage magnitude. */
+#define BUS_DEFAULT_V_MIN_REG 0.9 /**< @brief Default minimum regulated voltage magnitude. */
+#define BUS_DEFAULT_V_MAX_LIMIT 1.1 /**< @brief Default maximum voltage violation limit magnitude. */
+#define BUS_DEFAULT_V_MIN_LIMIT 0.9 /**< @brief Default minimum voltage violation limit magnitude. */
```

Comment:
No need to make distinction for default. Just keep BUS_DEFAULT_V_MIN and MAX. 
Also, "Default max/min voltage violation limit magnitude" sounds very strange in docstring. Please change this to "Default max/min voltage magnitude" (original form). The use of "voltage violation limit magnitude" appears in other parts of the code.